index.html

```diff
@@ -10,6 +10,72 @@
 </head>
 
 <body>
+    <main>
+        <div class='container'>
+            <div class="filler"></div>
+            <nav class='navbar'>
+                <a href=""><img src="images/Apple-Logo.png" alt=""></a>
+                <a href="">Store</a>
+                <a href="">Mac</a>
+                <a href="">iPod</a>
+                <a href="">iPhone</a>
+                <a href="">iPad</a>
+                <a href="">iTunes</a>
+                <a href="">Support</a>
+                <input type="text" name="search">
+            </nav>
+            <div class='text-container1'>
+                <img class="small"
+                    src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png"
+                    width="980" height="104" alt="What will your verse be?">
+                <img class="second"
+                    src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png"
+                    width="980" height="82"
+                    alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+                <div class="third"></div>
+            </div>
+        </div>
+        <div class="images-container">
+            <div class="img1"></div>
+            <div class="img2"></div>
+            <div class="img3"></div>
+            <div class="img4"></div>
+
+        </div>
+
+
+
+    </main>
+    <footer>
+        <div class="first-footer"><small>Shop the <a href="">Apple Online Store</a> (1-800-MY-APPLE), visit an <a
+                    href="">Apple Retail Store</a>, or
+                find a <a href="">reseller.</a></small></div>
+        <div class="second-footer">
+            <a href="">Site Map</a>
+            <a href="">Hot News</a>
+            <a href="">RSS Feeds</a>
+            <a href="">Media Info</a>
+            <a href="">Enviroment</a>
+            <a href="">Job Opportunities</a>
+            <a href="">Contact Us</a>
+            <a href="" style='border: none;'><img src="images/usa.png" alt=""></a>
```

Comment:
Same here with the inline styling, we can add a `.border-none` class and move it to `style.css`, if you want to reuse the border stlying in other components.